scripts/us_census/acs5yr/subject_tables/common/resolve_geo_id.py

```diff
@@ -0,0 +1,48 @@
+"""
+Resolving geoIds given the GEOID string from data,census.gov
+"""
+
+## The map between summary level and FIPS code is based on
+# https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html
+# A detailed note is available at https://mcdc.missouri.edu/geography/sumlevs/
+_SUMMARY_LEVELS_FIPS_CODE = {
+    '040': 2,
+    '050': 5,
+    '060': 10,
+    '140': 11,
+    '150': 12,
+    '160': 7,
+    '500': 4
+}
+
+
+def _convert_to_geoId(row):
```

Comment:
You are converting to DCID.  Maybe `convert_to_dcid`  (also please don't mix snake and camel case)

Please following docstring guidelines https://google.github.io/styleguide/pyguide.html#383-functions-and-methods